apps/exceptions.py

```diff
@@ -107,6 +107,14 @@ def exception_handler(exc, context):
     if not response:
         return unhandled_drf_exception_handler(exc, context)
 
+    if response.status_code == status.HTTP_400_BAD_REQUEST:
+        for field, detail in list(exc.detail.items()):
+            if isinstance(detail, dict):
+                keys = detail.keys()
+                for key in keys:
```

Comment:
Combine line 113 and 114 to one line

```suggestion
                for key in detail.keys():
```